### ISSUE TITLE
Using the extension makes the fan go noisy

### ISSUE THREAD

**Issue by agazso** [stephane-13/gnome-shell-extension-dim-background-windows#20]
First of all, thanks for the extension, it is great and I would definitely use it.

However I have an issue with it, when I turn it on the fans in my laptop go noisy. I checked the System Monitor app but I don't see any irregularity, so maybe the extension is doing something GPU related (which is not shown in the System Monitor)?

Anyone else out there who has similar issues?

I am using PopOS 22.04, which is based on Ubuntu 22.04 and uses Gnome 42. I have both an Intel integrated graphics card and an NVidia card and with both I get the same results.

**Comment by stephane-13** [comment]
Thank you for your report.

I never noticed a power usage increase on my computer, which is a desktop with a low-to-mid range graphic card (RX 6600). It's connected to a power meter, so I would immediately see some abnormal power usage. At least something that would correspond to a fan speed increase on a laptop, typically 10-20W or more.

Laptops usually have graphic cards with lower performance, but it shouldn't really matter for this extension.
The GLSL fragment shader used to dim the windows is very simple and shouldn't cause more than a 1% GPU usage, if not lower.

However, here are some options to verify if the extension is the source of a possible GPU usage spike on your system:
- The "sensors" command output (from the "lm-sensors" package installable with apt), which might indicate the GPU stats including its fan(s) speed
- The NVIDIA nvidia-smi tool if you use the proprietary driver (not sure about the nouveau driver)

Let me know if you can find something with those tools.

**Comment by agazso** [comment]
Thanks for the response!

I tried the tools that you suggested but could not found anything. However I have a laptop from Tuxedo and they provide a tool called [Tuxedo Control Center](https://www.tuxedocomputers.com/en/TUXEDO-Control-Center-TCC.tuxedo) and with that I could figure out that basically the fans very overly sensitive so I changed to a quieter profile without any downsides.

I leave this here so if someone runs into this issue can get some clues, but closing the issue anyways.
